x-pack/plugins/security_solution/common/endpoint/types.ts

```diff
@@ -41,14 +41,30 @@ type ImmutableMap<K, V> = ReadonlyMap<Immutable<K>, Immutable<V>>;
 type ImmutableSet<T> = ReadonlySet<Immutable<T>>;
 type ImmutableObject<T> = { readonly [K in keyof T]: Immutable<T[K]> };
 
+export interface EventStats {
+  /**
+   * The total number of related events (all events except process and alerts) that exist for a node.
```

Comment:
what is the reason for excluding alerts here? I'm wondering how this code will need to change if we start supporting events from sources other than the elastic endpoint.

Reply:
Yeah that's a good point. I'm keeping them separate because in the mocks we have a separate drop down/view into how many related alerts there are. I don't believe we'll be breaking the alerts down by "type" at the moment either.

@bkimmel is that accurate?